src/Pages/Login/Login.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { SEND_TO_SERVER } from '../../config';
+
+export default function Login() {
+  const { Kakao } = window;
+  let history = useHistory();
+
+  const KakaoLoginClickHandler = () => {
+    Kakao.Auth.login({
+      success: function (authObj) {
+        sendTokenToServer();
+      },
+    });
+  };
+
+  function sendTokenToServer() {
+    fetch(SEND_TO_SERVER, {
+      method: 'POST',
+      headers: {
+        Authorization: Kakao.Auth.getAccessToken(),
+      },
+    })
+      .then(res => res.json())
+      .then(res => {
+        if (res.message === 'SUCCESS') {
+          localStorage.setItem('access_token', res.token);
+          history.push('/');
+        } else if (res.message === 'INVALID_USER') {
+          const state = {
+            pathname: '/signup',
+            state: {
+              email: res.email,
+              id_number: res.id_number,
+            },
+          };
+          history.push(state);
+        }
+      });
```

Comment:
```suggestion
      .then(({ message, token, email, id_number }) => {
        if (message === 'SUCCESS') {
          localStorage.setItem('access_token', token);
          history.push('/');
        } else if (message === 'INVALID_USER') {
          history.push({
            pathname: '/signup',
            state: { email, id_number },
          });
        }
      });
```
- 함수 인자 자리에서 직접 구조분해 할당이 가능합니다
- 굳이 변수에 할당해줄 필요 없이 직접 넣는 것이 더 가독성 측면에서 좋아보이네요!
- 객체의 키와 값 자리에 들어오는 변수의 이름이 같을 경우 생략해줄 수 있습니다.

Reply:
오 감사합니다~ 바꾸겠습니다!